### PR TITLE
Framework: track stats on chunk loading.

### PR DESCRIPTION
**Pull request by mtias**
- Expose chunk name in ui state.
- Track waiting event for each chunk.

### CODE REVIEW

**Comment by rralian** [comment]
LGTM!

File: client/components/pulsing-dot/index.jsx [hunk 1]
```diff
@@ -12,6 +17,30 @@ var PulsingDot = React.createClass( {
...
+
+	componentWillReceiveProps: function( nextProps ) {
+		if ( nextProps.active === this.props.active ) {
+			return;
+		}
+
+		if ( nextProps.active ) {
+			this._loadingStatTimeout = setTimeout( () => {
+				analytics.mc.bumpStat( 'calypso_chunk_waiting', this.props.chunkName );
+			}, 400 );
```
Comment (ehg) [inline client/components/pulsing-dot/index.jsx]: We might want to define this as a constant, if we're using it elsewhere.

File: client/components/pulsing-dot/index.jsx [hunk 2]
```diff
@@ -12,6 +17,30 @@ var PulsingDot = React.createClass( {
...
+	},
+
+	componentWillReceiveProps: function( nextProps ) {
+		if ( nextProps.active === this.props.active ) {
+			return;
+		}
+
+		if ( nextProps.active ) {
+			this._loadingStatTimeout = setTimeout( () => {
+				analytics.mc.bumpStat( 'calypso_chunk_waiting', this.props.chunkName );
```
Comment (ehg) [inline client/components/pulsing-dot/index.jsx]: I think we prefer dashes for stat groups? Super minor thing though!

**Comment by ehg** [comment]
Two really minor things, so :+1:

File: client/components/pulsing-dot/index.jsx (hunk 2 above)
Comment (mtias) [inline client/components/pulsing-dot/index.jsx]: Checking with @apeatling it seems we do underscores which are then converted to spaces for display?

**Comment by mtias** [comment]
Thanks for the review!

**Pull request merged**